backend/app/services/users/add_birthdate.rb

```diff
@@ -0,0 +1,20 @@
+# frozen_string_literal: true
+
+module Users
+  class AddBirthdate < Core::Service
+    def call(birthdate:, user:)
+      @birthdate = birthdate
+      @user = user
+
+      add_birthdate
+    end
+
+    private
+
+    attr_reader :birthdate, :user
+
+    def add_birthdate
+      user.update(birthdate:) ? Success() : Failure()
```

Comment:
We need to handle both options. 
It's better to return something meaningful in `Failure`, for example `Failure(:user_not_updated)` and then check for this error in controller and return corresponding response